[AGM-2402] Update AdGuard for Linux (#673)
AdguardTeam/KnowledgeBase: docs/adguard-for-linux/installation.md

## Install AdGuard for Linux

> Before starting, you have to open the command line in your computer. On Linux, press `Ctrl+Alt+T`. On Mac, type `Terminal` in the search field.

Reviewer: ```suggestion
Open the command line on your computer. On Linux, press `Ctrl+Alt+T`. On Mac, type `Terminal` in the search field.
```

To install AdGuard, enter:

> Release

Reviewer: это команда для терминала?

Author: это горячие клавиши, переписала в кавычках

Reviewer: вопрос был про строку 12 со словом Release
а горячие клавиши и вводимое слово Terminal лучше выделить курсивом. см. редполитику о том, когда кавычки используем: https://www.notion.so/adguard/0df1a9bec9464fbf88bd813312ab0c26?pvs=4#25735cd772fd4083852762bc9d2ba57a

:::

## Initial setup

> AdGuard for Linux requires an [AdGuard license](https://adguard.com/license.html). If you don't have a license yet, you can log in or create an account to get a free 14-day trial. If you already have a license, log in to activate it. To do it, follow this step-by-step guide:

Reviewer: ```suggestion
AdGuard for Linux requires an [AdGuard license](https://adguard.com/license.html). If you don’t have a license yet, you can log in or create an account to get a free 14-day trial. If you already have a license, log in to activate it. To do it, follow this step-by-step guide:
```

> 1. After installation, enter

Reviewer: ```suggestion
1. After installation, enter:
```

>     adguard-cli activate

Reviewer: думаю, команды для терминала стоит выделять ``

and select the desired option:

![adguard-cli activate *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation1.png)

1. Once you have a license, follow the provided link to proceed with the activation:

![Activate license *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation2.png)

1. After activation, you can continue to work with the program:

![Successful activation *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation3.png)

> You can also reset your license by entering

Reviewer: ```suggestion
You can also reset your license by entering:
```

    adguard-cli reset-license

> or  view the license info by entering

Reviewer: ```suggestion
or view the license info by entering:
```

    adguard-cli license

1. Now you need to set up AdGuard CLI. To run the configuration wizard, enter:

    adguard-cli configure

![Setup *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation4.png)

The wizard will ask basic questions required for the initial setup.

> 1. After that, you can start the AdGuard protection by entering

Reviewer: ```suggestion
1. After that, you can start the AdGuard protection by entering:
```

    adguard-cli start

![Start protecton *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation5.png)

> 1. You can check the protection status using

Reviewer: ```suggestion
1. You can check the protection status using:
```

    adguard-cli status

> and stop it with

Reviewer: ```suggestion
and stop it with:
```

    adguard-cli stop

![Stop protecton *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation6.png)

> 1. To check the current configuration, enter

Reviewer: ```suggestion
1. To check the current configuration, enter:
```

    adguard-cli config show

![Current setup *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation7.png)

The path to the configuration file will also be displayed, which you can and should edit directly. It contains descriptions of all additional options.

As an alternative, you can use the commands:

    adguard-cli config get

> or

Reviewer: ```suggestion
or:
```

    ```

1. Then select the desired option:

    ![adguard-cli activate *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation1.png)

1. Once you have a license, follow the provided link to proceed with the activation:

    ![Activate license *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation2.png)

1. After activation, you can continue to work with the program:

    ![Successful activation *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation3.png)

1. You can also reset your license by entering:

    ```sh
    adguard-cli reset-license
    ```

    or  view the license info by entering

    ```sh
    adguard-cli license
    ```

1. Now you need to set up AdGuard CLI. To run the configuration wizard, enter:

    ```sh
    adguard-cli configure
    ```

    ![Setup *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation4.png)

    The wizard will ask basic questions required for the initial setup.

1. After that, you can start the AdGuard protection by entering

    ```sh
    adguard-cli start
    ```

    ![Start protecton *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/activation5.png)

>     1. You can check the protection status using:

Reviewer 2: If this is an item in the same list, it should not be indented.
